### DISCUSSION TITLE
What is "d option" in I-SIMPA?

### DISCUSSION THREAD

**Discussion by Shima-Mnz** [Ifsttar/I-Simpa#322] [General]
Hi,
When I import new scene and use “Meshing” in toolbar , sometimes I see this error:
"tetgen.exe : Hint: use -d option to detect all self-intersections."
What is "d option" ?
![WhatsApp Image 2022-07-05 at 2 50 37 PM](https://user-images.githubusercontent.com/106420309/177307156-9de26eb3-32a5-4972-b081-7d3fc7671d1b.jpeg)

**Comment by nicolas-f** [comment]
Hi,
It detects self-intersection in yoyr 3d model. I-simpa will use it if you check "test mesh topology" and highlight it in the 3d model in red.

https://wias-berlin.de/software/tetgen/1.5/doc/manual/manual005.html#sec36

https://i-simpa-wiki.readthedocs.io/fr/latest/code_configuration_meshing.html?highlight=tetgen#meshing
